README.md

```diff
@@ -41,7 +41,9 @@ Give a ⭐ if you like this project!
 
 This project is [MIT](./MIT.md) licensed.
 
-
+![](https://github.com/degisew/Vet-clinic-database/blob/query-multiple-tables/shoots/q1.PNG)
+![](https://github.com/degisew/Vet-clinic-database/blob/query-multiple-tables/shoots/q2.PNG)
+![](https://github.com/degisew/Vet-clinic-database/blob/query-multiple-tables/shoots/q3.PNG)
```

Comment:
hey there, ur project looks good but u need one more step to go further.
the links to your queries are broken so kindly fix them and try again or you can add a screen shot of your queries by using the snipping tool on you pc.

Reply:
Okay, thanks. I will fix them.